airflow/cli/commands/scheduler_command.py

```diff
@@ -34,12 +35,17 @@
 from airflow.utils.cli import process_subdir, setup_locations, setup_logging, sigint_handler, sigquit_handler
 from airflow.utils.scheduler_health import serve_health_check
 
+log = logging.getLogger(__name__)
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```
Maybe this? Usually I see it's called logger

Reply:
Could you please point to where the logger is named `logger`? The modules I've touched so far all named logger as `log` [example](https://github.com/apache/airflow/blob/73b90c48b1933b49086d34176527947bd727ec85/airflow/jobs/scheduler_job_runner.py#L780)

Reply:
yes. The example you shared is using `self.log` and not `log`. I believe this comes from extending the [LoggingMixin](https://github.com/apache/airflow/blob/main/airflow/utils/log/logging_mixin.py#L64) class which has the [log](https://github.com/apache/airflow/blob/main/airflow/utils/log/logging_mixin.py#L84) property.

What I meant is in general when using `logging.getLogger(__name__)`, python modules call it as `logger` across the projects I have seen in so far. One example in our project I could find is here https://github.com/apache/airflow/blob/73b90c48b1933b49086d34176527947bd727ec85/airflow/utils/log/file_task_handler.py#L46


But I think I have another suggestion here. Since this is a CLI command I guess we could use the [AirflowConsole](https://github.com/apache/airflow/blob/main/airflow/cli/simple_table.py#L39) here to print messages e.g. https://github.com/apache/airflow/blob/main/airflow/cli/commands/connection_command.py#L360

Reply:
Naming variables one way or another seems ultimately trivial, so I am perfectly okay with changing the variable name to `logger`.

On the other hand, I am not sure about using `AirflowConsole` for logging purposes. The code changes are located under the `cli` module, but they are ultimately meant to catch exceptions that came up within the inner logic of the scheduler, so I think it should be treated like a normal logger.

Another situation where I think using a logger is more appropriate is when the scheduler is run as a daemon (with `airflow scheduler -D`), in which case using the `AirflowConsole` will not log the exception.

Reply:
Naming on this is unfortunately pretty inconsistent in Airflow. Personally I much prefer `logger` but 🤷